Contributing.md

```diff
@@ -0,0 +1,59 @@
+# Contributing to DeliverEase
+
+Welcome to DeliverEase! We are excited to have you contribute to our project. DeliverEase is made using Django (Python) for the backend and HTML/CSS for the frontend. Your contributions can help us improve and enhance the platform, empowering delivery partners and connecting communities even more effectively.
```

Comment:
I wouldn't recommend adding the tech stack explicitly as that is really volatile with time.
cc: @Devansh-Singh-Parmar